ggml-weights on huggingface

gitbearflying opened the pull request.

sevagh commented:
Thanks!

sevagh commented:
Actually I noticed you've put a GPL 3.0 license on the weights in Huggingface, which isn't exactly appropriate. They're under a sort of unspecified/custom license: https://github.com/facebookresearch/demucs/issues/327#issuecomment-1134828611 - but they're definitely not GPL 3.0

gitbearflying commented:
> Actually I noticed you've put a GPL 3.0 license on the weights in Huggingface, which isn't exactly appropriate. They're under a sort of unspecified/custom license: [facebookresearch/demucs#327 (comment)](https://github.com/facebookresearch/demucs/issues/327#issuecomment-1134828611) - but they're definitely not GPL 3.0

I changed it to MIT, since this is what Intel did here https://huggingface.co/Intel/demucs-openvino/tree/main .
I also added a readme which mentions the source of the weights.
